gokart/target.py

```diff
@@ -102,7 +102,7 @@ def _dump(self, obj) -> None:
             self._processor.dump(obj, f)
 
     def _remove(self) -> None:
-        if self._target.exists():
+        if self._exists():
```

Comment:
@mski-iksm how about

```suggestion
        if self.exists():
```

(for all other occurrences)

because _exists() is inner method, exists() is better, I think.

Reply:
@mski-iksm ping?

Reply:
@Hi-king I thought self.exists() is better because it closes the relationship within this class without creating relationships with the super class. How do you think?

Reply:
@mski-iksm OK. I just concern about future implementation between exist() and _exist() will be skipped. But might be too early concern.

Reply:
@Hi-king I understood your concern  and also think that every class that inherit TargetOnKart have to do existence check is troublesome. I will move the existence check of _remove() to remove(). Thank you for the comment!